spec/codecs/json_lines_spec.rb

```diff
@@ -58,6 +58,48 @@
         insist { decoded } == true
       end
     end
+
+    context "when json could not be parsed" do
+
+      let(:message)    { "random_message" }
```

Comment:
you need to add a `\n` after the message to trigger the codec. Otherwise it will never enter the decode block. So make it  `let(:message)    { "random_message\n" }`